build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
@wolfgangmm No! We discussed this yesterday in the teleconf.
We agreed to one version number (e.g. `project.version`) number which is semver. Build scripts can be updated to use that.

Release candidates for 3.1.0 will report their version as 3.1.0. Extensions don't need to know the difference because of the short life-span of a RC. i.e. Extensions are never released against an RC they are only tested against RCs and releases of Extensions are against final eXist versions.